requirements_dev.txt

```diff
@@ -0,0 +1,18 @@
+# CPU Compatible Requirements
+-r requirements.txt
+
+# Dev:
+# Development, documentation, testing related dependencies.
+pytest
```

Comment:
```suggestion
pytest~=6.2
```

Reply:
Good catch. Yup. Better to version these. Gonna add it in.